src/app/about/Chapter_card.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Image from "next/image";
+
+interface ChapterCardProps {
+  image: string;
+  name: string;
+  description: string;
+}
+
+export default function chapter_card({
+  image,
+  name,
+  description,
+}: ChapterCardProps) {
+  return (
+    <div className="bg-dark  max-w-sm rounded-lg border  border-yellow-500 shadow dark:border-gray-700 dark:bg-gray-800">
+      <div className="flex items-center justify-center pt-5">
+        <Image src={image} alt="Chapter Logo" width={300} height={300} />
```

Comment:
I think alt attribute should be similar to the ones in Init_Team.tsx: alt={`${name} Chapter Logo`}